reana_job_controller/variables.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Variables for use with htcondorVC3."""
+
+import os
+
+MAX_JOB_RESTARTS = 3
```

Comment:
If these variables are used only in VC3 context, it would be good to name the file more specifically.  Examples:

- Do we need to have them as separate file? If not, consider moving into htcondorvc3 class.
- Shall they be reused for other VC3 backends than just HTCondor? If yes, leave them in separate file named like `vc3_config.py` or `vc3_variables.py`.
- We can always use `vc3_...` name prefix technique for other possible VC3-specific files such as job wrapper.  Currently it lives in `files/job_wrapper.sh`, it may be better to use something like `scripts/vc3/job_wrapper.sh`.

Reply:
@khurtado any thoughts on if we'd like to move these or prefix the naming?

Reply:
Looking into it, it seems we do not these 2 anymore:

https://github.com/reanahub/reana-job-controller/blob/0e7660967e28dd42b285e31b29cac7c6e0dbf18c/reana_job_controller/variables.py#L16-L17

So, I think we can simply remove `variables.py` and move `MAX_JOB_RESTARTS` below the import calls in `htcondorvc3_job_manager`